src/js/components/Anchor.js

```diff
@@ -43,6 +49,7 @@ const Anchor = props => {
 
 
 Anchor.propTypes = {
+  customIcon: PropTypes.string,
```

Comment:
Let's call this just "icon" instead of "customIcon". That will align with how "label" works.